<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>629</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="baseSize">
       <size>
        <width>700</width>
        <height>410</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="managePage">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string>品牌车管理</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolBox" name="toolBox">
            <property name="font">
             <font>
              <family>Gabriola</family>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>374</width>
               <height>422</height>
              </rect>
             </property>
             <attribute name="label">
              <string>出售车辆</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="4">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>万</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>品牌：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>厂家：</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="2">
               <widget class="QLineEdit" name="sellSumLineEdit"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>金额</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2" colspan="3">
               <widget class="QLabel" name="sellLastNumLabel">
                <property name="text">
                 <string>剩余数量：000</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>报价：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="4">
               <widget class="QComboBox" name="sellFactoryComboBox"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>数量：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3">
               <widget class="QLineEdit" name="sellPriceLineEdit"/>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="sellNumSpinBox"/>
              </item>
              <item row="1" column="1" colspan="4">
               <widget class="QComboBox" name="sellBrandComboBox"/>
              </item>
              <item row="4" column="3">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>万</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QPushButton" name="sellOkBtn">
                <property name="text">
                 <string>确定</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="3">
               <widget class="QPushButton" name="sellCancelBtn">
                <property name="text">
                 <string>取消</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>374</width>
               <height>422</height>
              </rect>
             </property>
             <attribute name="label">
              <string/>
             </attribute>
            </widget>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QListWidget" name="dailyList"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chartPage">
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>0</y>
          <width>401</width>
          <height>71</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>销售统计</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="factoryComboBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="updateBtn">
           <property name="text">
            <string>更新显示</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="manageMenu">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="separator"/>
    <addaction name="manageAction"/>
    <addaction name="charAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="passwordMenu">
    <property name="title">
     <string>修改密码</string>
    </property>
   </widget>
   <addaction name="manageMenu"/>
   <addaction name="passwordMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="manageAction">
   <property name="text">
    <string>品牌车管理</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="charAction">
   <property name="text">
    <string>销售统计</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
